test-site/src/ts/index.ts

```diff
@@ -8,7 +8,7 @@ import initDirectAnswers from './initDirectAnswers';
 
 const coreConfig: SearchConfig = {
   apiKey: process.env.API_KEY,
-  experienceKey: 'slanswers',
+  experienceKey: 'developer-support-test',
```

Comment:
Does this experience key impact all of the test site options? Like universal search, vertical search, etc.? If so, let's make sure everything still works

Reply:
I adjusted this experience to make sure everything still works for the ts page. 
Flagging that for the js page, in index.js, the experience key is 'rosetest' which I think no longer exists (at least isn't an experience for the two businesses with experiences mentioned in [this](https://docs.google.com/document/d/1EusosNnvHpT_J3RvTnz1IjnQBGQ_unnI7tfBJFoRzQA/edit?usp=sharing) doc)? So the js test site doesn't really work (says `invalid params. experienceKey: 'rosetest'` when you try most of the searches).

Reply:
I'm guessing we archived the rosetest experience at some point.

Reply:
Looks like for one of the checks (run linter), 'rosetest' is needed in order for it to pass - leaving it as is for now but can look into this more!